wls-ergebnismeldung-service/src/test/java/de/muenchen/oss/wahllokalsystem/ergebnismeldungservice/service/begruendung/BegruendungServiceSecurityTest.java

```diff
@@ -0,0 +1,146 @@
+package de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.service.begruendung;
+
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.ArgumentMatchers.notNull;
+import de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.MicroServiceApplication;
+import de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.TestConstants;
+import de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.configuration.Profiles;
+import de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.domain.BegruendungRepository;
+import de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.domain.Stapelart;
+import de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.rest.begruendung.BegruendungDTOMapper;
+import de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.utils.Authorities;
+import de.muenchen.oss.wahllokalsystem.wls.common.exception.TechnischeWlsException;
+import de.muenchen.oss.wahllokalsystem.wls.common.security.BezirkIDPermissionEvaluator;
+import de.muenchen.oss.wahllokalsystem.wls.common.testing.SecurityUtils;
+import java.util.stream.Stream;
+import lombok.val;
+import org.apache.commons.lang3.ArrayUtils;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.aggregator.ArgumentsAccessor;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.security.access.AccessDeniedException;
+import org.springframework.test.context.ActiveProfiles;
+
+@SpringBootTest(classes = MicroServiceApplication.class)
+@ActiveProfiles({ TestConstants.SPRING_TEST_PROFILE, Profiles.DUMMY_CLIENTS })
+class BegruendungServiceSecurityTest {
+
+    @MockBean
+    BezirkIDPermissionEvaluator bezirkIDPermissionEvaluator;
+
+    @Autowired
+    BegruendungService unitUnderTest;
+
+    @Autowired
+    BegruendungRepository begruendungRepository;
+
+    @Autowired
+    BegruendungDTOMapper begruendungDTOMapper;
+
+    @BeforeEach
+    void setup() {
+        SecurityUtils.runWith(Authorities.REPOSITORY_DELETE_BEGRUENDUNG);
+        begruendungRepository.deleteAll();
+    }
```

Comment:
ich bin hier etwas verwirrt mit den `beforeEach` und `afterEach` funtionen. (habe die für mich richtige bezeichnung auch schon ein zwei mal im review suggested, siehe controllerIntegrationTest)
für mich würde sinn machen 
```
@BeforeEach
void setup() {
    SecurityContextHolder.clearContext();
}
```
oder 
```
@AfterEach
void tearDown() {
    SecurityUtils.runWith(Authorities.REPOSITORY_DELETE_BEGRUENDUNG);
    begruendungRepository.deleteAll();
}
```` 
oder beides. hier ist aber oft eine Mischung und das habe ich auch bei anderen services gesehen. vielleicht sollten wir dazu nochmal ein issue aufmachen um das zu klären?

Reply:
Ich hab hier jetzt mal Variante 2 eingearbeitet. ich hab nichts dagegen wenn wir das kurz ansprechen, ist schließlich ne sache von ein paar min

Reply:
ich halte das nicht für sinnvoll, gerade da man hier kaum von einheitlich sprechen kann. z.B. in den AWerten und im Status wurde ebenfalls die Variante 2 gewählt. Ich würde das erstmal so lassen, ein DiskussionIssue dafür aufmachen und dann wenn eine Entscheidung steht, alles einheitlich refactoren

Reply:
#673